starknet/rust/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 use std::ffi::{c_char, CStr, CString};
 use cairo_lang_starknet::casm_contract_class::CasmContractClass;
 
+// why return type is mutable ?
```

Comment:
No particular reason, probably because `CString::into_raw` returns a mutable pointer. I should've used `CString::as_ptr`